fiftyone/core/utils.py

```diff
@@ -2828,9 +2828,6 @@ def get_multiprocessing_context():
         # subsequent usage of things like `multiprocessing.Queue()` will not
         # cause the default start method to switch to 'spawn'
         multiprocessing.set_start_method("fork", force=True)
-    elif sys.platform == "win32":
```

Comment:
I'm not sure who added this `win32` block in #5642, but I don't think it is needed and thus we should remove it because we shouldn't be force-ably overwriting the user's `multiprocessing` start method if they have manually configured one.

For context, the `darwin/linux` block immediately above this does use `force=True`, but **ONLY** if the user hasn't manually set a start method. And even then, it is setting the start method for a specific reason as explained in the comment, which doesn't apply to Windows.